samples/sqltransport/multi-instance-migration/sample.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Multi-Instance Mode to Bridge migration
+summary: Migration of SQL Server transport Multi-Instance Mode topology to Bridge
+reviewed: 2017-09-04
+component: SqlTransport
+related:
+ - transports/sql/deployment-options
+---
+
+
+## Prerequisites
+
+include: sql-prereq
+
+The databases created by this sample are `NsbSamplesSqlMultiInstanceReceiver3`, `NsbSamplesSqlMultiInstanceSender3`, `NsbSamplesSqlMultiInstanceReceiver4`, `NsbSamplesSqlMultiInstanceSender3` and `NsbSamplesSqlMultiInstanceBridge`.
+
+Ensure [Distributed Transaction Coordinator (DTC)](https://msdn.microsoft.com/en-us/library/ms684146.aspx) is running. It can be started from the command line by running `net start msdtc`.
+
+
+## Running the project
+
+ 1. Start all projects.
+ 1. Hit enter in Sender V3 and V4 consoles window to send messages.
+
+
+## Verifying that the sample works correctly
+
+ 1. The Receiver displays information that an order was submitted.
+ 1. The Sender displays information that the order was accepted.
+
+
+## Code walk-through
+
+This sample contains the following projects:
+
+ * Sender.V3 - A console application responsible for sending the initial `ClientOrder` message and processing the follow-up `ClientOrderAccepted` message.
+ * Receiver.V3 - A console application responsible for processing the order message.
```

Comment:
A console application running/using NServiceBus Version 6 and SQL Server transport Version 3.